ui/src/pages/View.js

```diff
@@ -164,8 +194,11 @@ class View extends Component {
   }
 
   renderField(labelMessage, value) {
+    if (value == null || value === '') {
+      return null;
+    }
     return (
-      <div>
+      <div key={labelMessage.id}>
```

Comment:
What is this key used for? It feels a little awkward to me to use the message ID as a key, and it might mean that they're not always unique: for example, if someone had multiple Twitter accounts, the Twitter message ID would be used as the key for two of these.